ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -183,3 +183,17 @@ def gumbel(loc=0.0, scale=1.0, size=None):
     u = ivy.random_uniform(low=0.0, high=1.0, shape=size, dtype="float64")
     x = loc - scale * ivy.log(-ivy.log(u))
     return x
+
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def triangular(left, mode, right, size=None):
+    if left > mode or mode > right or left == right:
+        raise ValueError("left < mode < right is not being followed")
```

Comment:
May be replace this with `IvyValueError`

Reply:
replaced ValueError with IvyValueError